packages/compartment-mapper/src/link.js

```diff
@@ -19,13 +20,12 @@
  *   LinkResult,
  *   ModuleDescriptor,
  *   ParseFn,
- *   ParseFnAsync,
  *   ParserForLanguage,
  *   ParserImplementation,
  *   ShouldDeferError,
  * } from './types.js'
+ * @import {ERef} from '@endo/eventual-send'
```

Comment:
If the type consuming `ERef` is present in an _external_ type, then `@endop/eventual-send` it should be a production dependency of `@endo/compartment-mapper`.  If not, it _should_ be added as a dev dependency anyway, but might be OK to just leave it as-is given we're in a monorepo.